lib/index.js

```diff
@@ -1,8 +1,11 @@
 const core = require('./core');
+const config = require('./config')
+
 module.exports = core;
 module.exports.Api = require('./api');
 module.exports.Core = core;
 module.exports.Extensions = require('./extensions');
+module.exports.getInClusterConfig = config.getInCluster
```

Comment:
`module.exports.config = require(./config);`

I'm anticipating we'll add more config-like options, so it would be good to start out with them namespaced under `.config`.